### PR TITLE
MISPDEV-26: Integrate AdminTool and propertySynchronizer utilities into the source repository

### PR DESCRIPTION
**Pull request by raits**
JIRA ticket: https://nordic-institute.atlassian.net/browse/MISPDEV-26

### CODE REVIEW

**Review by petkivim — CHANGES_REQUESTED** [review]
Currently, the Admin Tool related documentation seems to be available here:

https://github.com/nordic-institute/misp2/blob/main/docs/misp2_installation_manual_18.04.md#61-administration-of-misp2-administrator-accounts-from-the-command-line

Maybe consider moving it to `utils/admin-tool/manual.md` and referencing it from the installation manual and Admin Tool readme (`utils/admin-tool/readme.md`). In that way, the approach would be consistent with the Property Synchronizer. 

Also, the Property Synchronizer manual (`utils/property-synchronizer/manual.md`) isn't currently referenced from any other document in the repo (except readme). Is there a place in the documentation where the tool could be mentioned, e.g., installation guide or manager guide?

File: utils/property-synchronizer/manual.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+| ![European Union / European Regional Development Fund / Investing in your future](../../docs/img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
+
+# MISP2 Installation and Configuration Guide
```
Comment (petkivim) [inline utils/property-synchronizer/manual.md:4]: This should probably be:

> MISP2 Property Synchronizer Manual

File: utils/property-synchronizer/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# PropertySynchronizer
```
Comment (petkivim) [inline utils/property-synchronizer/readme.md:1]: This should probably be:

> Property Synchronizer

Comment (raits) [reply utils/property-synchronizer/readme.md]: Fixed.

File: utils/property-synchronizer/manual.md (hunk 1 above)
Comment (raits) [reply utils/property-synchronizer/manual.md]: Fixed.

**Comment by raits** [comment]
> Currently, the Admin Tool related documentation seems to be available here:
> 
> https://github.com/nordic-institute/misp2/blob/main/docs/misp2_installation_manual_18.04.md#61-administration-of-misp2-administrator-accounts-from-the-command-line
> 
> Maybe consider moving it to `utils/admin-tool/manual.md` and referencing it from the installation manual and Admin Tool readme (`utils/admin-tool/readme.md`). Also, it should be referenced from `docker-dev/readme.md`. In that way, the approach would be consistent with the Property Synchronizer.
> 
> Also, the Property Synchronizer manual (`utils/property-synchronizer/manual.md`) isn't currently referenced from any other document in the repo (except readme). Is there a place in the documentation where the tool could be mentioned, e.g., installation guide or manager guide?

Migrated the Admin Tool manual to its own file and added links.

I would leave the Property Synchronizer as it is. It is more of an implementation detail and is used only by the installation script. There should be no need for the user to do anything with it, but good to have the manual for developers that might need to understand what it is supposed to do.

**Review by petkivim — APPROVED** [review]
Looks good to me.

**Pull request merged**